menu.js

```diff
@@ -1,4 +1,5 @@
-function clickFunction('') {
+// eslint-disable-next-line no-unused-vars
```

Comment:
We couldn't find out how to fix the linter error `no-unused-vars`, so we added this line of code to pass the linters, if you have any idea why the linter failed, please do let us know, and we will fix it. :)